tests/api_connexion/schemas/test_dag_schema.py

```diff
@@ -122,11 +123,11 @@ def test_serialize(self):
             'is_paused': None,
             'is_subdag': False,
             'orientation': 'LR',
-            'owners': [],
+            'owner': '',
             'params': {'foo': 1},
             'schedule_interval': {'__type': 'TimeDelta', 'days': 1, 'seconds': 0, 'microseconds': 0},
             'start_date': '2020-06-19T00:00:00+00:00',
-            'tags': None,
+            'tags': ["example1", "example2"],
```

Comment:
We should return objects to make it more future-proof and extensible. for example, if we want to add a color, we can do so without breaking changes.

Reply:
Previously, the endpoint returns tags object and that made it to output empty objects {} because in the DAG object, tags is a list of strings unlike DagModel where tags is an object.

I think, since this endpoint only outputs data and doesn't add, we will not worry about adding anything else. I have removed the connection between the schema with DagSchema where it inherited DagSchema which defined tags. 

Now the schema is standing alone and not connected with other schemas, and only for the Dagdetails endpoint which only outputs data.

What do you think?

Reply:
Fixed